sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -283,6 +283,13 @@ export interface BlobExistsOptions extends CommonOptions {
    * @memberof BlobExistsOptions
    */
   customerProvidedKey?: CpkInfo;
+  /**
+   * Conditions to meet.
+   *
+   * @type {BlobRequestConditions}
+   * @memberof BlobExistsOptions
+   */
+  conditions?: BlobRequestConditions;
```

Comment:
Consider to add test, if this is fix brought by this change.

Reply:
Irrelevant to this feature. We somehow missed it.

Reply:
Added a test case.